at_secondary/at_persistence_secondary_server/lib/src/log/commitlog/at_commit_log.dart

```diff
@@ -31,7 +32,7 @@ class AtCommitLog implements AtLogType {
     // The private: and privatekey: are not synced. so return -1.
     if (!key.startsWith('public:__') &&
         (key.startsWith(RegExp(
-                'private:|privatekey:|public:_|public:signing_publickey')) ||
+                'private:|privatekey:|public:_|public:signing_publickey|${statsNotificationId.toLowerCase()}')) ||
```

Comment:
Maybe make the whole regex case-insensitive rather than depending on the case of the things we're checking against. https://api.dart.dev/stable/2.18.2/dart-core/RegExp/isCaseSensitive.html

Reply:
set the caseSensitive to false in Regex.